packages/react/.storybook/main.ts

```diff
@@ -39,7 +39,7 @@ const config: StorybookConfig = {
 				// }
 			},
 			optimizeDeps: {
-				exclude: ['@carbon/telemetry']
+				exclude: ['@ibm/telemetry-js']
```

Comment:
@theiliad I figured this maybe needed to be updated but not sure what the considerations of this are so just FYI, I can rollback if that's not the case